customer-config/checkout-config/che.plulist.v2.json

```diff
@@ -312,7 +312,7 @@
           "description": "A unique id",
           "type": "string",
           "minLength": 1,
-          "maxLength": 20
+          "maxLength": 40
```

Comment:
There is a limit in the POS database that is set to 20, so this cannot be modified